pixeltable/env.py

```diff
@@ -394,12 +429,21 @@ def _set_up(self, echo: bool = False, reinit_db: bool = False) -> None:
         self._set_up_runtime()
         self.log_to_stdout(False)
 
+    @property
+    def default_system_db_url(self):
+        if self._db_server is None:
+            return self._dbms.default_system_db_url()
+        else:
+            return self._db_server.get_uri(database='postgres', driver='psycopg')
```

Comment:
Why not just use dbms.default_system_db_url here as well?

Reply:
fixed dbms. default_system_db_url can work w/o need of dbserver just need to replace databsase in url that ie has with 'postgres'